js/src/data/actions.js

```diff
@@ -1276,3 +1276,34 @@ export function* receiveAdsRecommendations(
 		recommendationTypes,
 	};
 }
+
+/**
+ * Action containing detailed error information.
+ *
+ * @generator
+ * @param {string} errorSlot - Unique key identifying the error (e.g., field name or error code).
+ * @param {Error|Object|null} error - The original error object or additional error details.
+ * @return {{type: string, errorSlot: string, errorType: string, error: (Error|Object|null)}}
+ *          Redux action with type `TYPES.RECEIVE_DETAILED_ERROR` and the provided error payload.
+ */
+export function* receiveDetailedError( errorSlot, error ) {
```

Comment:
```suggestion
export function* receiveDetailedError( slot, error ) {
```

@ankitrox We can use the `slot` name as parameter instead of `errorSlot`